pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
```

Comment:
This feels really common.  In fact, it feels so common I think I'd like to be able to project this from the pod template via a downward API env var.  Or an automatic env var.  Everyone having to do this feels wrong.